### PR TITLE
15.1.0

### PR DESCRIPTION
**Pull request by zpao**
As we said in #6803, we're going to ship 15.1.0 as it stands today.

I'm mostly looking to make sure the wording in the changelog is fine. In particular, the description here for the Perf tooling is really short and could use a little bit more, maybe even a followup blog post? (cc @gaearon)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+## 15.1.0 (May 19, 2016)
+
+### React
+- Ensure we're using the latest `object-assign`, which has protection against a non-spec compliant native `Object.assign`. ([@zpao](https://github.com/zpao) in [#6681](https://github.com/facebook/react/pull/6681))
```
Comment (spicyj) [inline CHANGELOG.md]: nit: "non–spec compliant" with an en dash or maybe "non-spec-compliant"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+- Add a new warning to communicate that `props` objects passed to `createElement` must be plain objects. ([@richardscarrott](https://github.com/richardscarrott) in [#6134](https://github.com/facebook/react/pull/6134))
+- Fix a batching bug resulting in some lifecycle methods incorrectly being called multiple times. ([@spicyj](https://github.com/spicyj) in [#6650](https://github.com/facebook/react/pull/6650))
+
+### ReactDOM
```
Comment (spicyj) [inline CHANGELOG.md]: nit: React DOM

**Comment by gaearon** [comment]
I can do a separate small blog post on changes to ReactPerf if you’d like.

**Comment by facebook-github-bot-2** [comment]
@zpao updated the pull request.

**Pull request merged**